crates/nu-cli/src/completions/completer.rs

```diff
@@ -137,6 +137,7 @@ impl NuCompleter {
                             let is_passthrough_command = spans
                                 .first()
                                 .filter(|content| *content == &String::from("sudo"))
+                                .filter(|content| *content == &String::from("doas"))
```

Comment:
Filtering twice with different predicates would return an empty iterator.
So let's combine them with an or (`||`)

```suggestion
                                .filter(|content| *content == &String::from("sudo") || *content == &String::from("doas"))
```

While we are at it we could make the types a bit nicer (by dereferencing `String` to `&str`)

```suggestion
                                .filter(|content| content.deref() == "sudo" || content.deref() == "doas")
```